utils/minimax_voice_clone.py

```
@@ -0,0 +1,287 @@
+
+    # ------------------------------------------------------------------
+    # 组合便捷方法: upload + create 一步完成
+    # ------------------------------------------------------------------
+    async def clone_voice(
+        self,
+        audio_buffer: io.BytesIO,
+        filename: str,
+        prefix: str,
+        language: str = "zh",
+    ) -> str:
+        """上传音频并创建音色（组合两步），返回 voice_id。"""
```

Ruff (0.15.6)

[warning] Docstring contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF002)